fedocal/__init__.py

```diff
@@ -621,7 +621,12 @@ def ical_calendar_meeting(meeting_id):
     if not meeting:
         return flask.abort(404)
     ical = vobject.iCalendar()
-    fedocallib.add_meeting_to_vcal(ical, meeting)
+    fedocallib.add_meeting_to_vcal(
+        ical, meeting,
+        enable_reminder=flask.request.args.get('reminder') == '1',
```

Comment:
Not sure we should support only ``1`` here.

Might be an idea: https://github.com/fedora-infra/pkgdb2/blob/master/pkgdb2/api/packages.py#L1048-L1050